### PR TITLE
Feature fix checkstyle UI

### PR DESCRIPTION
**Pull request by danielaSantaCruz**
- Complier views
- Controller views
- Converter views
- Metadata views
- Model views
- Text-extractor views
- Utilities view

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java [hunk 1]
```diff
@@ -33,21 +33,28 @@
     private MetaDataInterface metaData;
     private AudioConverterInterface audio;
     private TextExtractorInterface textExtractor;
+    private final int alignLabelStyle = 0;
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/view/MainInterface.java:36]: are these variables or constants? if they are constants they should be in capital letters.

File: logs/propertieslogs.log [hunk 2]
```diff
@@ -954,4 +954,4 @@
 [INFO ] 2021-04-14 17:07:34.760 [http-nio-8080-exec-4] ExtractMetadata - finish
 [INFO ] 2021-04-14 17:07:34.760 [http-nio-8080-exec-4] ExtractMetadata - start
 [INFO ] 2021-04-14 17:07:34.760 [http-nio-8080-exec-4] ExtractMetadata - Execute Try
-[INFO ] 2021-04-14 17:07:34.763 [http-nio-8080-exec-4] ExtractMetadata - finish
+[INFO ] 2021-04-14 17:07:34.763 [http-nio-8080-exec-4] ExtractMetadata - finish
```
Comment (JRojas999) [inline logs/propertieslogs.log]: logs should not be uploaded

**Review by edsonarios — APPROVED** [review]
Good job

File: src/main/java/org/fundacion/jala/converter/view/MainInterface.java (hunk 1 above)
Comment (danielaSantaCruz) [reply src/main/java/org/fundacion/jala/converter/view/MainInterface.java:36]: In a code convention, we decided that constants would be written in lowercase :)

**Review by Raymundogs — APPROVED** [review]

File: logs/propertieslogs.log (hunk 2 above)
Comment (danielaSantaCruz) [reply logs/propertieslogs.log]: Already done :)

**Review by JRojas999 — APPROVED** [review]

File: logs/propertieslogs.log (hunk 2 above)
Comment (JRojas999) [reply logs/propertieslogs.log]: Good job

**Pull request merged**